### PR TITLE
Fix rinkeby base tokens

### LINKED ISSUE #1123: Rinkeby base tokens is broken
**Description**
You cannot select a token. They have mainnet addresses so if you click on them, nothing get's selected

**How to Reproduce**
1. Connect to Rinkeby
2. Open the token selector
2. Click on DAI, USDC or USDT

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Closes #1123 

Rinkeby base tokens was broken. This PR makes sure the tokens use Rinkeby address (not mainnet one, that was the issue)

Additionally, adds also the broken images for the base tokens:
image

As a bonus, fixes the image for WETH that was broken in Rinkeby for a long time! (already in production)

image

As u see, now is fixed:
image

### CODE REVIEW

File: src/custom/utils/rinkeby/constants.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Token } from '@uniswap/sdk-core'
+import { SupportedChainId as ChainId } from 'constants/chains'
+
+export const DAI_RINKEBY = new Token(ChainId.RINKEBY, '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea', 18, 'DAI', 'DAI')
+
+export const USDT_RINKEBY = new Token(
+  ChainId.RINKEBY,
+  '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa',
```
Comment (alfetopito) [inline src/custom/utils/rinkeby/constants.ts:8]: Both addresses are the same

screenshot_2021-07-29_11-06-24

**Review by alfetopito — CHANGES_REQUESTED** [review]
Opsie, little mistake on the addresses there

File: src/custom/utils/rinkeby/constants.ts (hunk 1 above)
Comment (anxolin) [reply src/custom/utils/rinkeby/constants.ts:8]: 🙈

**Comment by anxolin** [comment]
Fixed the issue with USDT, the address of USDT was wrong:
image

**Comment by anxolin** [comment]
@alfetopito u can review after the merge

**Comment by elena-zh** [comment]
LGTM!

**Pull request merged**